services/galley/src/Galley/API/Internal.hs

```diff
@@ -543,8 +553,35 @@ internalDeleteFederationDomainH ::
   Domain ::: JSON ->
   Sem r Response
 internalDeleteFederationDomainH (domain ::: _) = do
+  -- We have to send the same event twice.
+  -- Once before and once after defederation work.
+  -- https://wearezeta.atlassian.net/wiki/spaces/ENGINEERIN/pages/809238539/Use+case+Stopping+to+federate+with+a+domain
+  void sendNotifications
   deleteFederationDomain domain
+  void sendNotifications
   pure (empty & setStatus status200)
+  where
+    sendNotifications = do
+      maxPage <- inputs $ fromRange . currentFanoutLimit . _options -- This is based on the limits in removeIfLargeFanout
+      page <- embedClient $ paginate selectAllMembers (paramsP LocalQuorum () maxPage)
+      sendNotificationPage page
```

Comment:
Shouldn't we void here instead of the call sites?

Reply:
I've moved the void in front of this call.